bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/robot/RRMapFileParser.java

```diff
@@ -299,6 +300,16 @@ public String toString() {
         return sw.toString();
     }
 
+    private void printAreaDetails(ArrayList<float[]> areas, PrintWriter pw) {
+        areas.forEach(area -> {
+            pw.printf("\tArea details:");
```

Comment:
If there are more areas does it make sense to include some information about what area these details are about. With your change you would get several generic "Area details" printed next to each other.

Reply:
Improved the text slightly to make it area coordinates to give hint on what the numbers are.

Reply:
btw, in each case, just before this is called, it prints out the area type.
```
No Go zones:	2
	Area coordinates:	356.0	218.0	340.0	218.0	340.0	207.0	356.0	207.0
	Area coordinates:	329.0	222.0	321.0	222.0	321.0	205.0	329.0	205.0
Walls:	1
	Area coordinates:	274.0	363.0	273.0	291.0
Zones:	0

```